public/assets/ts/admin/image-store.ts

```diff
@@ -86,7 +86,7 @@ class ImageStoreFrontend {
       body: formData,
     };
 
-    const response = await fetch("/api/image/upload", options);
+    const response = await fetch("/api/admin/image/upload", options);
```

Comment:
from what I can see, these endpoints aren't served at /api/admin/...?
i can only see /api/.... (from server.ts)

Reply:
yea it looks like the frontend calls this endpoint (and all the other /api/admin/... endpoint and it fails). could you look into this? or did I miss something

Reply:
ooh good point, fixed!

(I remember testing this?? apparently not thoroughly enough :clown_face:)